deploy/olm-catalog/keycloak-operator/7.0.1/keycloak-operator.v7.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
```

Comment:
I believe that we can list this as a stage 4 operator. We have install, upgrades, backup and monitoring

Reply:
+1 for "Deep Insights".